brain_brew/build_tasks/overrides/headers_override.py

```diff
@@ -17,25 +17,39 @@ def task_name(cls) -> str:
     def yamale_schema(cls) -> str:
         return f'''\
             deck_description_html_file: str(required=False)
+            crowdanki_uuid: str(required=False)
+            name: str(required=False)
         '''
 
     @dataclass
     class Representation(RepresentationBase):
         deck_description_html_file: Optional[str]
+        crowdanki_uuid: Optional[str]
+        name: Optional[str]
```

Comment:
I think that we need default values (`None`) here, if we want to allow overriding only some of `deck_description`, `crowdanki_uuid` and `name` at the same time.

```suggestion
        deck_description_html_file: Optional[str] = None
        crowdanki_uuid: Optional[str] = None
        name: Optional[str] = None
```

Reply:
I don't believe so, since the actual overriding function individually check if each value is truthy. Have you experienced that populating one in the recipe, and not others, will null out the others?

Reply:
Overriding one in the recipe, but not the others causes BrainBrew to crash:

```
TypeError: __init__() missing 2 required positional arguments: 'crowdanki_uuid' and 'name'
```

Reply:
Huh, I guess you're right. I never made that default value in the original since it was the only member of the Override class. Pushed a change that should fix it :+1: